rules.md

```diff
@@ -354,17 +354,17 @@ Your claim will then be processed by an admin.
 
 ### 2.3.5 Processing claims
 
-An Admin will process new claims, assessing them based on the rules for the type of output you’ve produced and whether the work completes the Task adequately as it was defined.
+All claims are processed by Admins.  Claims are assessed based on the rules for the type of output produced, and whether the work completes the Task adequately as it was defined.  
```

Comment:
Ambiguity? Does this need to be clearer on which Admin processes the claim)?

Reply:
Again the issue seems to be the document structure (see other comments).  This probably needs to be addressed in a separate Rule Change.

Reply:
Yes, I agree, this is probably a matter for a future PR addressing some of the existing grammatical & other idiosyncrasies, rather than the subsatnce of your changes. We can refer back to it when addressing those at a later date.